### ISSUE TITLE
@ symbol causing error.

### ISSUE THREAD

**Issue by aarondfrancis** [lob/lob-php#83]
I'm currently validating a metric crap-ton of county data and wanted to give you some feedback on an error I've gotten. (This is all public data, so I haven't blanked anything out.)

Given this data:

```json
{
    "owner_id": "194249",
    "owner_name": "BAHR D E",
    "owner_care_of": null,
    "owner_addr_1": "@ DE ZIMMERMANN",
    "owner_addr_2": "P O BOX 23",
    "owner_addr_3": null,
    "owner_city": "SANTA FE",
    "owner_state": "TX",
    "owner_zip": "77517",
    "owner_zip_4": "0023",
}
```

I get the following error:

```
[2018-01-08 17:16:29] local.ERROR: fopen( DE ZIMMERMANN): failed to open stream: No such file or directory {"exception":"[object] (ErrorException(code: 0): fopen( DE ZIMMERMANN): failed to open stream: No such file or directory at /project/vendor/lob/lob-php/src/Lob/Resource.php:200)
```

I assume the county is using `@` to represent "Care Of", but it's clearly mucking things up.

Can you add a safeguard check to `Resource.php`?

**Comment by dmlittle** [comment]
Hi @aarondfrancis, thanks for reporting this issue! 

It looks like the Lob library assumes strings starting with an `@` symbol to represent local files to be included in the request ([link](https://github.com/lob/lob-php/blob/29db02611aefb8b9603c84fc643bf25a95bbbde0/src/Lob/Resource.php#L173-L175)). This is certainly a bug as `"@ DE ZIMMERMANN"` is not a file. I'll take a look into fixing this issue in the upcoming sprint.

**Comment by dmlittle** [comment]
@aarondfrancis a new version of the PHP library is now available with this fix included (v2.1.2)
